bridge/src/lib.rs

```diff
@@ -85,9 +85,9 @@ pub mod pallet {
 	pub trait Config: frame_system::Config + sygma_access_segregator::Config {
 		type RuntimeEvent: From<Event<Self>> + IsType<<Self as frame_system::Config>::RuntimeEvent>;
 
-		/// Bridge transfer reserve account
+		/// Bridge transfer reserve accounts mapping with designated assets
 		#[pallet::constant]
-		type TransferReserveAccount: Get<Self::AccountId>;
```

Comment:
How about declaring it as `BTreeMap::<AssetId, Self::AccountId>`? It looks like we don't need to iterate two lists in `get_token_reserved_account` anymore